lib/resources/example_json/soarian_examples_procedure.rb

```diff
@@ -0,0 +1,170 @@
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi011"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "This section represents data charted for a single or set of encounter’s procedures performed. It does not represent a comprehensive procedure list."
+                    }
+                ]
+            }
+        },
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "papi029"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Information provided is from 2015-06-01 through 2015-06-28 inclusive."
+                    }
+                ]
+            }
+        },
+        {
+            "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+            "valueExtension": {
+                "url": "http://cerner.hs.fhir.com/StructureDefinition/bundle-result-messages",
+                "extension": [
+                    {
+                        "url": "Severity",
+                        "valueString": "info"
+                    },
+                    {
+                        "url": "Code",
+                        "valueString": "fsp001"
+                    },
+                    {
+                        "url": "Message",
+                        "valueString": "Date parameter values passed with request were: date=>=2015-06-01T11:18:00&date=&lt;2015-06-29T11:18:00"
+                    }
+                ]
+            }
+        }
+    ],
+    "type": "searchset",
+    "link": [
+        {
+            "relation": "self",
+            "url": "base-url/123abc/Procedure?date=%3E%3D2015-06-01T11%3A18%3A00&date=%3C2015-06-29T11%3A18%3A00&_format=json&patientId=7568F2F3-FC76-4185-A540-1DB56331A387"
+        }
+    ],
+    "entry": [
+        {
+            "resource": {
+                "resourceType": "Procedure",
+                "text": {
+                    "status": "additional",
+                    "div": "&lt;li ccdRefId=\'ZCCDProcedureList.CCDProcedureList.1503\'>Completed Bronchoscopy, on 6/22/2015 12:00 AM&lt;/li>"
+                },
+                "identifier": [
+                    {
+                        "use": "official",
```

Comment:
Its an identifier assigned by Soarian, FHIR cardinality allows the “system” and “type” as optional.